benchmark/benchmark_cv2.py

```diff
@@ -0,0 +1,77 @@
+import time
```

Comment:
lets remove this, I'm planning a entire benchmarking overhaul for the week after I finish classes. I want to make things more robust